Ginger/Ginger/Actions/ActionEditPages/ActWebSmartSyncEditPage.xaml.cs

```diff
@@ -89,69 +89,69 @@ private void NotifyLocatorVisibilityChanged()
-            SetPanelVisibility(TxtMatch_Pnl, ActNewSmartSync.eSyncOperation.TextMatches);
-            SetPanelVisibility(UrlMatches_Pnl, ActNewSmartSync.eSyncOperation.UrlMatches);
-            SetPanelVisibility(AttributeMatches_Pnl, ActNewSmartSync.eSyncOperation.AttributeMatches);
+            SetPanelVisibility(TxtMatch_Pnl, ActWebSmartSync.eSyncOperation.TextMatches);
+            SetPanelVisibility(UrlMatches_Pnl, ActWebSmartSync.eSyncOperation.UrlMatches);
+            SetPanelVisibility(AttributeMatches_Pnl, ActWebSmartSync.eSyncOperation.AttributeMatches);
             setOperationDescription();
         }
 
         void setOperationDescription()
         {
             switch (ActionNameComboBox.SelectedValue)
             {
-                case ActNewSmartSync.eSyncOperation.ElementIsVisible:
+                case ActWebSmartSync.eSyncOperation.ElementIsVisible:
                     xOperationDescription.Text = "Verifies if an element is present on both the webpage's structure (DOM) and visible to the user.";
                     break;
-                case ActNewSmartSync.eSyncOperation.ElementExists:
+                case ActWebSmartSync.eSyncOperation.ElementExists:
                     xOperationDescription.Text = "Checks if an element exists on the webpage, regardless of whether it is visible.";
                     break;
-                case ActNewSmartSync.eSyncOperation.AlertIsPresent:
+                case ActWebSmartSync.eSyncOperation.AlertIsPresent:
                     xOperationDescription.Text = "Waits until an alert message pops up on the webpage. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.ElementIsSelected:
+                case ActWebSmartSync.eSyncOperation.ElementIsSelected:
                     xOperationDescription.Text = "Confirms if a form element like a checkbox or radio button is currently selected. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.PageHasBeenLoaded:
+                case ActWebSmartSync.eSyncOperation.PageHasBeenLoaded:
                     xOperationDescription.Text = "Ensures that the entire webpage has been loaded and is ready for interaction. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.ElementToBeClickable:
+                case ActWebSmartSync.eSyncOperation.ElementToBeClickable:
                     xOperationDescription.Text = "Waits until an element is both visible and enabled, indicating that it can be clicked. ";
                     break;
-                case ActNewSmartSync.eSyncOperation.TextMatches:
-                    xOperationDescription.Text = "Waits until the text of an element matches a specified pattern. ";
+                case ActWebSmartSync.eSyncOperation.TextMatches:
+                    xOperationDescription.Text = "Waits until the text of an element matches a specified pattern. Input Text is case Sensitive.";
```

Comment:
change it to "Waits until the text of an element matches a specified pattern. Input Text is case-sensitive and does the contains search."